scripts/test_wheel.sh

```diff
@@ -94,7 +103,40 @@ fi
 
 test -d deps/env || USE_MINIMAL="$USE_MINIMAL" ./scripts/install_python_toolchain.sh
 source deps/env/bin/activate
-pip install target/turicreate-*.whl
+
+bdist_wheels=($(ls target/turicreate-*.whl))
+if [[ ${#bdist_wheels[@]} -eq 1 ]]; then
```

Comment:
I'm not really understanding this whole if/elif/else code block. Please add some comments.

Reply:
added

```
# make_wheel can build 1 or 2 wheels, namely full or minimal
# we need to find the wheel we need to install and test
bdist_wheels=($(ls target/turicreate-*.whl))
```